src/main/scala/ai/privado/languageEngine/javascript/tagger/collection/CollectionTagger.scala

```diff
@@ -136,6 +143,45 @@ class CollectionTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCp
 
     tagDirectSources(cpg, builder, collectionMethodsCache.l, collectionRuleInfo)
     tagDerivedSources(cpg, builder, collectionMethodsCache.l, collectionRuleInfo)
+
+    /*
+    Handle the case of DataElement present in url
+    app.get(
+      "/api/v1/messages/:fiscalcode/:id/:senderEmail?", /* <===  */
+      GetMessage(
+        config,
+        serviceModel,
+        messageModel,
+        messageStatusModel));
+
+     */
+
+    callMap.foreach(callEntry => {
+
+      methodUrlMap.get(callEntry._1) match
+        case Some(endPoint) =>
+          val params = endPoint
+            .stripPrefix("\"")
+            .stripPrefix("'")
```

Comment:
could be double quotes or backticks as well. we can just strip any non alpha character here ?

Reply:
I guess, if its a backticks that is already handled when adding to `methodUrlMap`

But anyways let me replace nonAlpha characters